openfl-tutorials/experimental/workflow/Privacy_Meter/requirements_privacy_meter.txt

```diff
@@ -6,3 +6,9 @@ pillow
 scikit-learn
 torch==2.3.1
 torchvision==0.18.1
+ray==2.9.2
+dill==0.3.6
+metaflow==2.7.15
+tabulate==0.9.0
+nbformat==5.10.4
+nbdev==2.3.12
```

Comment:
I suggest to keep those separate in the `workflow_interface_requirements.txt`:
https://github.com/securefederatedai/openfl/blob/develop/openfl-tutorials/experimental/workflow/workflow_interface_requirements.txt#L8

Reply:
removing this the user has to install two requirement.txt file for running the experiment, one from **workflow** directory and other from **Privacy_meter** directory.. so to make it single file installation, i have moved the required dependencies to privary_meter/requirement.txt file..

Reply:
made it into separate requirements file